src/Media/UploadMedia.php

```diff
@@ -54,12 +54,18 @@ private function addFiles(HasMedia $model, string $type, array $files, array &$f
 
     private function addFile(HasMedia $model, string $type, array &$files_order, $file)
     {
+        $locale = null;
```

Comment:
ik zou locale meegeven als hidden input in de view zodat deze berekening niet hoeft.